src/ceph-volume/ceph_volume/devices/lvm/zap.py

```diff
@@ -52,34 +52,29 @@ def wipefs(path):
     * ``CEPH_VOLUME_WIPEFS_INTERVAL``: Defaults to 5
 
     """
-    tries = str_to_int(
-        os.environ.get('CEPH_VOLUME_WIPEFS_TRIES', 8)
-    )
-    interval = str_to_int(
-        os.environ.get('CEPH_VOLUME_WIPEFS_INTERVAL', 5)
-    )
-
-    for trying in range(tries):
-        stdout, stderr, exit_code = process.call([
+    tries = str_to_int(os.environ.get('CEPH_VOLUME_WIPEFS_TRIES', 8)) + 1
+    interval = str_to_int(os.environ.get('CEPH_VOLUME_WIPEFS_INTERVAL', 5))
+
+    for attempt in range(1, tries):
+        _, _, exit_code = process.call([
```

Comment:
I generally try to discourage people from mixing logical changes from stuff like formatting changes or, in this case, typing annotations. It's fine to do it in one PR, but it can be confusing to do that in one commit. However, I won't block this PR over that.